hw/ip/entropy_src/dv/cov/entropy_src_UNR.vRefine

```diff
@@ -0,0 +1,2033 @@
ression coverage of operators in various conditions and assignments.;	set_expr_coverable_statements -procassign -event_control -misc;	;	// Enables scoring of Verilog modules compiled with -v/-y or -libcell option but continues to;	// disable the scoring of Verilog modules defined with the 'celldefine compiler directive.;	set_libcell_scoring;	;	// Enables scoring of SystemVerilog continuous assignments, which is by disabled by default.;	set_assign_scoring;	;	// Scores branches together with block coverage.;	set_branch_scoring;	;	// Scores statements within a block.;	set_statement_scoring;	;	// Enables Toggle scoring and reporting of SystemVerilog enumerations and multidimensional static;	// arrays , vectors, packed union, modport and generate blocks.;	set_toggle_scoring -sv_enum enable_mda -sv_struct_with_enum -sv_modport -sv_mda 16 -sv_mda_of_struct -sv_generate -sv_packed_union;	;	// Enable toggle coverage only on ports.;	set_toggle_portsonly;	;	// Enable scoring of FSM arcs (state transitions).;	// TODO: re-enable this setting, temp disable due to #12544;	// set_fsm_arc_scoring;	;	// Include X-&gt;1|0 for toggle coverage collection. #10332;	set_toggle_includex;	;	// For ternary operator in default SOP mode;	set_expr_scoring -vlog_short_circuit;;// enable coverage on dut and below;;select_coverage -befts -module ${DUT_TOP}...;;// Black-box pre-verified IPs from coverage collection.;deselect_coverage -betfs -module pins_if;deselect_coverage -betfs -module clk_rst_if;deselect_coverage -betfs -module prim_alert_sender...;deselect_coverage -betfs -module prim_alert_receiver...;deselect_coverage -betfs -module prim_count...;deselect_coverage -betfs -module prim_esc_sender...;deselect_coverage -betfs -module prim_esc_receiver...;deselect_coverage -betfs -module prim_onehot_check...;deselect_coverage -betfs -module prim_prince...;deselect_coverage -betfs -module prim_lfsr...;;;// Black-box DV CDC module.;deselect_coverage -betfs -module prim_cdc_rand_delay;;// csr_assert_fpv is an auto-generated csr read assertion module. So only assertion coverage is;// meaningful to collect.;deselect_coverage -betf -module *csr_assert_fpv...;select_coverage -assert -module *csr_assert_fpv;;// Only enable assertion coverage;deselect_coverage -betf -module *tlul_assert...;select_coverage -assert -module *tlul_assert;;// Only collect toggle coverage on the DUT and the black-boxed IP (above) ports.;deselect_coverage -toggle -module ${DUT_TOP}...;select_coverage -toggle -module ${DUT_TOP};;select_coverage -toggle -module prim_alert_sender;select_coverage -toggle -module prim_alert_receiver;select_coverage -toggle -module prim_count;select_coverage -toggle -module prim_esc_sender;select_coverage -toggle -module prim_esc_receiver;select_coverage -toggle -module prim_onehot_check;select_coverage -toggle -module prim_prince;select_coverage -toggle -module prim_lfsr;" path="/wdc/proj/cto/top/work/martin.lueker-boden/projects/opentitan/hw/dv/tools/xcelium/cover.ccf"></ccf-file>
```

Comment:
can you remove section `ucm-files`, `ccf-files` and `cache-map` (at the bottom)? This is the example.

https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/spi_host/dv/cov/spi_host_unr_excl.vRefine?q=vrefine&ss=opentitan%2Fopentitan

Reply:
Thanks for the suggestion Weicai.   Done.  I also noticed that I left the copyright comment out, so I've added that as well.